src/data/components.json

```diff
@@ -886,6 +886,32 @@
         "notAvailable": false
       }
     },
+    "Content group — banner": {
+      "url": "./content-group-banner",
+      "description": "Content group — banner is designed as a subtle callout, intended for tangential information or announcements that are supplementary to the page's main focus.",
+      "storybook": {
+        "webcomponents": "http://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-content-group-banner--default",
```

Comment:
I may be mistaken here, but I didn't think we usually link to Canary storybooks? We should link the the stable one when the release goes out? @JennySanchez cc @shixiedesign